packages/itwinui-react/jest.config.js

```diff
@@ -209,11 +209,11 @@ module.exports = {
 
   // An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
 
-  // testPathIgnorePatterns: [
-
-  //   "\\\\node_modules\\\\"
-
-  // ],
+  testPathIgnorePatterns: [
+    '\\\\node_modules\\\\',
+    '\\\\esm\\\\',
+    '\\\\cjs\\\\',
```

Comment:
Just curious, what is the reason for the backslashes? I'm guessing in L213 for example becomes `\\node_modules\\`, but not sure why the backslashes are needed.

Reply:
i actually could not even get it to work with that. changing it to `/` works. can you confirm on windows? go inside `packages/itwinui-react` and run `yarn dev` and `yarn test` in parallel terminals

Reply:
For me on Windows, it worked for both: `\\\\` and `/`

Reply:
ok, lets go with `/` then